test/Benchmarks/mlp_kernel.mlir

```diff
@@ -0,0 +1,42 @@
+// RUN: tpp-opt %s -map-linalg-to-tpp \
+// RUN:            -main-closure -pre-bufferization -pack-matmul="block-factors=2,2" -loop-invariant-code-motion -canonicalize -undo-main-closure \
```

Comment:
I would also drop `main-closure`, `undo-main-closure` and `loop-invariant-code-motion`.

Reply:
This specific test does not work without `-pre-bufferization`, I haven't looked why yet. #142